### PR TITLE
Compliance Api Group

### PR DESCRIPTION
**Pull request by avkos**

### CODE REVIEW

**Review by danforbes — APPROVED** [review]
Looks good 👍🏻 Please review [my changes](https://github.com/ChainSafe/web3-circle-libs/pull/62/commits/b3f054e0996a7edb5ea4490139d65b72195af31f) 🙏🏻

File: packages/web3-circle-sdk/src/MonitoredTokensApi.ts [hunk 1]
```diff
@@ -51,29 +51,19 @@ export class MonitoredTokensApi extends BaseApi {
    * Delete monitored tokens
    * https://developers.circle.com/api-reference/w3s/programmable-wallets/delete-monitored-tokens
    * @param params the parameters for the delete monitored token request
-   * @returns a boolean that indicates if the monitored tokens were deleted
+   * @returns void
```
Comment (luu-alex) [inline packages/web3-circle-sdk/src/MonitoredTokensApi.ts:54]:
```suggestion
   * @returns Promise<void>
```

**Review by luu-alex — APPROVED** [review]

**Comment by luu-alex** [comment]
LGMT, just return signature change

File: packages/web3-circle-sdk/src/FaucetApi.ts [hunk 2]
```diff
@@ -6,11 +6,9 @@ export class FaucetApi extends BaseApi {
    * Request testnet tokens for your wallet.
    * https://developers.circle.com/api-reference/w3s/programmable-wallets/request-testnet-tokens
    * @param params the parameters for the faucet request
-   * @returns a boolean that indicates if the faucet request was successful
+   * @returns void
```
Comment (jdevcs) [inline packages/web3-circle-sdk/src/FaucetApi.ts:9]:
```suggestion
 * @returns Promise<void> - A promise that resolves when the request is complete.
```

Comment (jdevcs) [reply packages/web3-circle-sdk/src/FaucetApi.ts:9]: and same for other instances in this PR

**Review by jdevcs — COMMENTED** [review]
lgtm, only few minor changes suggested.

File: packages/web3-circle-sdk/src/FaucetApi.ts (hunk 2 above)
Comment (danforbes) [reply packages/web3-circle-sdk/src/FaucetApi.ts:9]: For consistency, this should read `@returns a promise that resolves when the request is complete`

File: packages/web3-circle-sdk/src/MonitoredTokensApi.ts (hunk 1 above)
Comment (danforbes) [reply packages/web3-circle-sdk/src/MonitoredTokensApi.ts:54]: See my comment below https://github.com/ChainSafe/web3-circle-libs/pull/62#discussion_r1880296556

**Pull request closed without merging**